package.json

```diff
@@ -44,6 +45,11 @@
     ]
   },
   "devDependencies": {
-    "@types/styled-components": "^5.1.26"
+    "@reduxjs/toolkit": "^1.9.1",
```

Comment:
Toolkit jest spoko, już nawet twórcy polecają go używać w dokumnetacji. Ogólnie logika redux jest całkiem spoko tylko nie używasz tam slices, a to w sumie największy feature z toolkita

Reply:
przegadamy to na callu